src/podman.scss

```diff
@@ -1,5 +1,6 @@
 @use "ct-card.scss";
 @use "page.scss";
+@import "global-variables";
```

Comment:
What is this file and what do we need it for? Does it define `$pf-global--breakpoint--md`?

Reply:
nvm, just tested it and indeed it defines these variables

Reply:
Yes, exactly.